Tests/ApolloTests/HTTPTransportTests.swift

```diff
@@ -266,3 +313,11 @@ extension HTTPTransportTests: HTTPNetworkTransportRetryDelegate {
     }
   }
 }
+
+// MARK: - HTTPNetworkTransportGraphQLErrorDelegate
+
+extension HTTPTransportTests: HTTPNetworkTransportGraphQLErrorDelegate {
+  func networkTransport(_ networkTransport: HTTPNetworkTransport, receivedGraphQLErrors errors: [GraphQLError], retryHandler: @escaping (Bool) -> Void) {
+    self.graphQlErrors = errors
```

Comment:
doesn't the retry handler need to be called here?

Reply:
Doh 🤦‍♂️
Will be fixed